curriculum/js2/5.js

```diff
@@ -5,7 +5,8 @@
  */
 
 const solution = (a) => {
-  return 0;
+  if (!a.length) return '';
```

Comment:
same comment as previous function

Reply:
Same as my response to previous one(js2/4.js).